<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListStepWidget</class>
 <widget class="QWidget" name="TaskListStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>110</height>
   </rect>
  </property>
  <widget class="QLabel" name="stepIconLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>140</width>
     <height>10</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>stepNameLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="borderLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>140</width>
     <height>110</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
